compiler/pipes/filter-only-actually-used.cpp

```diff
@@ -171,7 +171,8 @@ IdMap<FunctionPtr> calc_actually_used_having_call_edges(std::vector<FunctionAndE
       fun->is_main_function() ||
       fun->type == FunctionData::func_class_holder || // classes should be carried along the pipeline
       (fun->is_extern() && vk::any_of_equal(fun->name, "wait", "make_clone")) ||
-      fun->kphp_lib_export;
+      fun->kphp_lib_export ||
+      fun->local_name() == ClassData::NAME_OF_TO_STRING;
```

Comment:
What I don't like here, is that every function will be searched for the "$$" substring (inside `local_name()` call).
Maybe, prevent this at least by doing in such a way:
```
(fun->modifiers.is_instance() && fun->local_name() == ClassData::NAME_OF_TO_STRING)
```

Reply:
Done.